packages/fx-core/src/plugins/resource/aad/utils/common.ts

```diff
@@ -63,4 +64,30 @@ export class Utils {
     const tenantId: string = (tokenObject as any).tid;
     return tenantId;
   }
+
+  public static getAndMoveInput(ctx: PluginContext): boolean {
```

Comment:
I cannot guess what this function is used for without go through the code? Can we provide a better naming for it?